### PR TITLE
Fixed header and added btn type

### PR DESCRIPTION
**Pull request by ASGladchenko**
Fixed Header, 
Added to btn style-type green,red
Create FileField

### CODE REVIEW

File: client/src/components/file-field/index.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+            if (!isExtensionAccept) {
+              form.setFieldError(name, `${errors('extensions')} ${accept}`);
+              form.setFieldTouched(name, true, false);
+
+              return;
+            }
+          }
+
+          if (maxSize && file.size / (1024 * 1024) > maxSize) {
```
Comment (rpkyrych) [inline client/src/components/file-field/index.tsx:47]: I'd extract both those if conditions to a separate functions

**Review by rpkyrych — APPROVED** [review]

File: client/src/components/input/index.tsx [hunk 2]
```diff
@@ -94,6 +99,21 @@ export const Input = ({
             />
           )}
 
+          {type === 'file' && (
+            <>
+              <input
+                type="file"
```
Comment (Wolshebnik) [inline client/src/components/input/index.tsx:105]: sort

File: client/src/components/input/index.tsx [hunk 3]
```diff
@@ -94,6 +99,21 @@ export const Input = ({
...
+                ref={ref}
+                name={props.name}
+                accept={props.accept}
+                className={styles.fileType}
+                onChange={onChange}
```
Comment (Wolshebnik) [inline client/src/components/input/index.tsx:110]: sort

File: client/src/hooks/types.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+export interface IUseWidowWidthProps {
+  width: number;
+  isMobile: boolean;
+  isNotMobile: boolean;
+  isTablet: boolean;
+  isLaptop: boolean;
+  isDesktop: boolean;
+  isNotDesktop: boolean;
+  isDesktopXL: boolean;
```
Comment (Wolshebnik) [inline client/src/hooks/types.ts:9]: sort

**Review by Kyslytsia — APPROVED** [review]

File: client/src/components/file-field/index.tsx (hunk 1 above)
Comment (ASGladchenko) [reply client/src/components/file-field/index.tsx:47]: In this case, putting it into a separate function will create a large number of bolerplate code and additional handlers and typing.

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**